src/app/DTT/abn/index.js

```diff
@@ -61,6 +61,16 @@ const Write = ({
   const dispatchBeInformation = useDispatch()
   const onNewMsg = compose(dispatchBeInformation, newMsg)
 
+  const fieldBackgroundColor = equals(clientId)('lojing')
```

Comment:
we cannot check for clientId or any other product related checks like this. We need to set a default colour and set of colour for lojing in the theme probably. Let's change it after the demo on Monday.